packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -534,6 +534,7 @@ The Authenticator object defines the following properties:
 | `created`     | String (ISO-8601)                                               | Timestamp when the Authenticator was created                                   |
 | `id`          | String                                                          | A unique identifier for the Authenticator                                                |
 | `key`         | String                                                          | A human-readable string that identifies the Authenticator                                |
+| `status`      | `ACTIVE`,`INACTIVE`                                             | Status of the Authenticator                                |
```

Comment:
Any other statuses?

Reply:
Here's the answer from @SamSanjabi-okta: 

https://okta.slack.com/archives/C9D0TLFQ9/p1630447255069300?thread_ts=1630447011.068300&cid=C9D0TLFQ9